vendor/src/github.com/vbatts/tar-split/tar/common/utf8.go

```diff
@@ -0,0 +1,22 @@
+package common
+
+// IsValidUtf8String checks for in valid UTF-8 characters
+func IsValidUtf8String(s string) bool {
+	return InvalidUtf8Index([]byte(s)) == -1
+}
+
+// IsValidUtf8Btyes checks for in valid UTF-8 characters
+func IsValidUtf8Btyes(b []byte) bool {
+	return InvalidUtf8Index(b) == -1
+}
+
+// InvalidUtf8Index returns the offset of the first invalid UTF-8 character.
+// Default is to return -1 for a wholly valid sequence.
+func InvalidUtf8Index(b []byte) int {
```

Comment:
oh shit. I thought there was such a thing, but did not find it in my searches...  I'll update.